### ISSUE TITLE
Cannot install via go get

### ISSUE THREAD

**Issue by k0pernikus** [dnephin/dobi#8]
Trying to install dobi via `go get`, I run into the error:

```
$ go get github.com/dnephin/dobi
# github.com/dnephin/dobi/cmd
/srv/go/src/github.com/dnephin/dobi/cmd/dobi.go:44: unknown cobra.Command field 'TraverseChildCommands' in struct literal
/srv/go/src/github.com/dnephin/dobi/cmd/dobi.go:45: unknown cobra.Command field 'Args' in struct literal
```

I'm using go 1.6:

```
$ go version
go version go1.6.2 linux/amd64
```

I am running Ubuntu16.04.

**Comment by dnephin** [comment]
Ah, I guess it is not go gettable at this time because my PRs on cobra haven't been merged yet.

I need to update the readme to say to use `git clone` instead.

The dependency versions are in the `glide.lock`:  https://github.com/dnephin/dobi/blob/master/glide.lock

Thanks for letting me know!

**Comment by dnephin** [comment]
This should be fixed e4ec29bfe5a8789770bb63df7c5f6cafeb277a65 , now in master, and docs are published.

Let me know if you have any problems with the new instructions.
